.lintstagedrc.json

```diff
@@ -0,0 +1,3 @@
+{
+    "*.ts": "pnpm dprint fmt"
```

Comment:
https://www.npmjs.com/package/lint-staged#task-concurrency also seems to imply that we should be printing files that are different, not actually formatting. Should this actually be:

```suggestion
    "*": "pnpm dprint check --list-different"
```

? Or will `*` actually pass in all files?

Reply:
> Is this a strict requirement?

No, no. Might do a `*`. I see that I will add `--allow-no-files` for the case when there are no files which can be formatted by the current plugins. Else `dprint` ends with a non-zero exit code.

> we should be printing files that are different, not actually formatting

To format is fine. `lint-staged` will add the modifications to the index to include them in the commit. Switching to `check --list-different` would fail the commit if any of the staged files is not correctly formatted. I'd rather keep writing them directly. 

We only have a single task, so there is not concurrency.